sources/packages/api/src-sql/SINValidations/Create-sin-validations.sql

```diff
@@ -0,0 +1,64 @@
+CREATE TABLE IF NOT EXISTS sims.sin_validations (
+  id SERIAL PRIMARY KEY,
+  date_sent TIMESTAMP WITH TIME ZONE,
+  date_received TIMESTAMP WITH TIME ZONE,
+  file_sent VARCHAR(50),
+  file_received VARCHAR(50),
+  given_name_sent VARCHAR(50),
+  surname_sent VARCHAR(50),
```

Comment:
I think in my application, we are using `last_name`

Reply:
or is this because, in file its surname?

Reply:
yes its because we use surname in the file